drivers/pcie/endpoint/pcie_ep_bcm_iproc.c

```diff
@@ -238,6 +240,155 @@ static int iproc_pcie_raise_irq(struct device *dev,
 	return ret;
 }
 
+static int iproc_pcie_register_reset_cb(struct device *dev,
+					enum pcie_reset reset,
+					void (*fn)(void *), void *arg)
```

Comment:
We usually use `typedef`s for callbacks in Zephyr

Reply:
I see. Will change accordingly.

Reply:
Used `typedef` for reset interrupt callback. Please check now.